src/pattern-library/components/PlaygroundApp.js

```diff
@@ -70,7 +71,6 @@ export default function PlaygroundApp({
               'border-transparent': activeRoute?.route !== route.route,
             })}
             href={route.route.toString()}
-            onClick={e => navigate(e, route.route)}
```

Comment:
This is no longer needed, as clicks on all `<a>` elements are now intercepted and handled client-side, depending on the URL.